RELEASES.md

```diff
@@ -0,0 +1,93 @@
+## Release Cadence and Support Policy ##
+
+This document describes the release cadence for runc as well as outlining the
+support policy for old release branches. Historically, despite runc being the
+most widely used Linux container runtime, our release schedule has been very
+ad-hoc and has resulted in very long periods of time between minor releases,
+causing issues for downstreams that wanted particular features.
+
+### Semantic Versioning ###
+
+runc uses [Semantic Versioning][semver] for releases. However, our
+compatibility policy only applies to the runc binary. We will make a
+best-effort attempt to reduce the impact to users that make direct use of
+`github.com/opencontainers/runc/libcontainer` as a Go library, but we do not
```

Comment:
nit: I'd rephrase this, as "a Go library" is probably not a universally understood term.

Something similar to this maybe:
```suggestion
Go packages prefixed with `github.com/opencontainers/runc`, but we do not
```